tools/install.sh

```diff
@@ -65,15 +67,15 @@ function _omb_install_readargs {
           install_prefix=$arg
         else
           install_opts+=:error
-          printf 'install (oh-my-bash): %s\n' "$RED$BOLD[Error]$NORMAL ${RED}the option argument for '--prefix' is empty.$NORMAL" >&2
+          printf 'install (oh-my-bash): %s\n' "$RED${BOLD}[Error]$NORMAL ${RED}the option argument for '--prefix' is empty.$NORMAL" >&2
```

Comment:
This is a workaround for Zsh, but this script is intended to be a Bash script, and there is already a check for the Bash version.

Reply:
Thanks. I've made the change and have updated the commit message accordingly.

Reply:
Thanks for updating!

I actually would like to make `disable=SC1087` a global setting for the shellcheck. The construct `$var[xxx]` is interpreted as an array expansion only in Zsh. It should never be an array expansion in POSIX sh and all the other POSIX-like shells. I naively think it's not so confusing to the users, so I'd like to turn `SC1087` off globally if possible. It is cleaner than putting `disable=SC1087` on every line. What do you think?

Reply:
Sure thing. I think that would be much simpler as well!